Scripts/freight_lem.py

```diff
@@ -0,0 +1,161 @@
+                                  list(purposes), args.specify_commodity_names))
+    
+    ass_model.prepare_freight_network(costdata["car_cost"], purps_to_assign)
+    impedance = ass_model.freight_network.assign()
+    impedance["toll"] = {mode: numpy.zeros([len(zone_numbers), len(zone_numbers)]) 
+                         for mode in ("truck", "freight_train", "ship")}
+    impedance["channel"] = {"ship" : numpy.zeros([len(zone_numbers), len(zone_numbers)])}
+    del impedance["cost"]
+    impedance = {mode: {mtx_type: impedance[mtx_type][mode] for mtx_type in impedance
+                        if mode in impedance[mtx_type]}
+                        for mode in ("truck", "freight_train", "ship")}
+    
+    matrix_counter = args.first_matrix_id
+    for purpose, purpose_value in purposes.items():
+        log.info(f"Calculating demand for purpose: {purpose}")
+        commodity_costs = costdata["freight"][commodity_conversion[purpose]]
+        costs = {"truck": {}, "freight_train": {}, "ship": {}}
+        costs["truck"]["cost"] = calc_road_cost(commodity_costs,
+                                                impedance["truck"])
+        costs["freight_train"]["cost"] = calc_rail_cost(commodity_costs,
+                                                        impedance["freight_train"])
+        costs["ship"]["cost"] = calc_ship_cost(commodity_costs,
+                                               impedance["ship"])
+        demand = purpose_value.calc_traffic(costs, purpose)
+        for mode in demand:
+            ass_model.freight_network.set_matrix(mode, demand[mode])
+            if purpose not in args.specify_commodity_names:
+                continue
+            # Explicitly save commodity
+            matrix_id = f"mf{matrix_counter}"
+            matrix_name = f"{purpose}_{mode}"
+            matrix_type = "demand"
+            ass_model._create_matrix(matrix_id, matrix_name, matrix_type, overwrite=True)
+            matrix_counter += 1
+            ass_model.freight_network.emme_matrices.update(
+                {matrix_name: {matrix_type: matrix_id}})
+            ass_model.freight_network.set_matrix(matrix_name, demand[mode])
+            with resultmatrices.open("freight_demand", "vrk", zone_numbers, m="a") as mtx:
+                mtx[matrix_name] = demand[mode]
+        if purpose in args.specify_commodity_names:
+            ass_model.freight_network.save_network_volumes(purpose)
+            matrix_counter += 7 # Shift to next 10 digit
+        ass_model.freight_network.output_traversal_matrix(resultdata.path)
+        aux_demand = transform_traversal_data(resultdata.path, zone_numbers)
+    log.info("Simulation ready.")
+
+if __name__ == "__main__":
+    parser = ArgumentParser(epilog="Freight lem-model-system entry point script.")
+    config = utils.config.read_from_file()
+    config["SAVE_EMME_MATRICES"] = True
+    config["SPECIFY_COMMODITY_NAMES"] = ["marita"]
```

Comment:
If you really want to have this as the default, could you move it to `dev-config.json`?